### PR TITLE
Build CLI from Docker instead of source codes, add alpine and jdk17 based Docker image

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

File: docs/en/contribution/release-java-agent.md [hunk 1]
```diff
@@ -276,7 +276,7 @@ export NAME=skywalking-java-agent
 export HUB=apache
 export TAG=$SW_VERSION
 
-make docker.push.java8 docker.push.java11
+make docker.push.alpine docker.push.java8 docker.push.java11
```
Comment (wu-sheng) [inline docs/en/contribution/release-java-agent.md:279]:
```suggestion
make docker.push.alpine docker.push.java8 docker.push.java11 docker.push.java17
```
How about this? As 17 is already LTS.

Comment (kezhenxu94) [reply docs/en/contribution/release-java-agent.md]: Sure, done

**Review by wu-sheng — APPROVED** [review]
I noticed 8.8.0-alphine is updated, let's make *-jdk17 available too.

https://hub.docker.com/r/apache/skywalking-java-agent/tags

**Comment by kezhenxu94** [comment]
> I noticed 8.8.0-alphine is updated, let's make *-jdk17 available too.
> 
> https://hub.docker.com/r/apache/skywalking-java-agent/tags

Pushed

**Pull request merged**